src/plugin/report-plugin/Listener/ReportListener.ts

```diff
@@ -145,6 +145,23 @@ export default class ReportListener {
             return;
         }
 
+        // Only allow Hotline members to react
+        const hotline = this.client.guilds.get('204100839806205953');
+            
+        try {
+            const member = hotline.members.get(userId);
+            if (!member) {
+                return;
+            }
+
+            // Only allow members who are in the Member role.
+            if (member.roles.indexOf('531617261077790720') == -1) {
```

Comment:
I suggest doing here `if (!member.roles.includes('531617261077790720'))` for better readability